lightning/restore/restore.go

```diff
@@ -1344,21 +1400,18 @@ func setSessionConcurrencyVars(ctx context.Context, db *sql.DB, dsn config.DBSto
 // DoChecksum do checksum for tables.
 // table should be in <db>.<table>, format.  e.g. foo.bar
 func DoChecksum(ctx context.Context, db *sql.DB, table string) (*RemoteChecksum, error) {
-	ori, err := increaseGCLifeTime(ctx, db)
-	if err != nil {
-		return nil, errors.Trace(err)
+	var err error
+	manager, ok := ctx.Value(&gcLifeTimeKey).(*gcLifeTimeManager)
+	if !ok {
+		return nil, errors.New("No gcLifeTimeManager found in context, check context initialization")
+	}
+
+	if err = manager.addOneJob(ctx, db); err != nil {
```

Comment:
how about we rename `addOneJob` to `start` and rename `removeOneJob` to `done`.

Reply:
ok start/done or start/finish? I prefer later

Reply:
I prefer `done`, but `finish` is fine

Reply:
sorry I don't support renaming now. `start` seems the job action is IN this function, but in fact `addOneJob` is intend to do prepare jobs, increase counter, e.g. Job action (checksum) is after the `addOneJob` call